adafruit_midi/midi_message.py

```diff
@@ -155,14 +175,17 @@ def _search_eom_status(cls, buf, eom_status, msgstartidx, msgendidxplusone, endi
         return (msgendidxplusone, good_termination, bad_termination)
 
     @classmethod
-    def _match_message_status(cls, buf, msgstartidx, msgendidxplusone, endidx):
+    def _match_message_status(
+        cls, buf: bytearray, msgstartidx: int, msgendidxplusone: int, endidx: int
+    ) -> Tuple[Optional[Any], bool, bool, bool, bool, int]:
         msgclass = None
         status = buf[msgstartidx]
         known_msg = False
         complete_msg = False
         bad_termination = False
 
         # Rummage through our list looking for a status match
+        assert msgclass is not None
```

Comment:
I think this assert would raise an exception any time this function were called. 

`msgclass` is initialized as None just a few lines above here and nothing seems to change it before this assert.